docs/victorialogs/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ according to [these docs](https://docs.victoriametrics.com/victorialogs/quicksta
 
 ## tip
 
+* BUGFIX: [querying](https://docs.victoriametrics.com/victorialogs/querying/#live-tailing): fix `/select/logsql/tail` requests being rejected by `-search.maxQueryTimeRange` when the query relied on the tail window for the time filter. Live tailing now continues to work with that flag enabled.
```

Comment:
```suggestion
* BUGFIX: [querying](https://docs.victoriametrics.com/victorialogs/querying/#live-tailing): fix `/select/logsql/tail` requests being rejected by `-search.maxQueryTimeRange` when the query relied on the tail window for the time filter. Live tailing now continues to work with that flag enabled. See [#760](https://github.com/VictoriaMetrics/VictoriaLogs/issues/760).
```

@func25 , please always add links to the related issues in the description line at the CHANGELOG.md . These links serve the following purposes:
- They allow CHANGELOG readers quickly investigating the details of every change.
- They are used for updating the corresponding issues with the human-readable information during the release process.